src/tools/cargotest/Cargo.toml

```diff
@@ -1,5 +1,5 @@
 [package]
-name = "cargotest"
+name = "cargotest2"
```

Comment:
Oh this conflicted with the `cargotest` package in Cargo itself, so I figured it'd be easiest to rename this here